tests/test_wo_with_no_signature.py

```diff
@@ -0,0 +1,257 @@
+#! /usr/bin/env python3
```

Comment:
Add a test case for using requesterSignature

Reply:
Done